bcgov/cas-registration
# Feat/333 apply emission categories

Applies an emission category (or categories) on save to a report_emission record based on the rules from the Emission Summary excel sheet in our SME channel files.

DB:
- New emission_category & emission_category_mapping tables
- m2m relationship added to report_emission table
- Initial data for the new tables

API:
- Added new function (apply_emission_categories) to the report_activity_save_service to apply categories on save to the report_emission table

FE:
- Added an API call to the backend call the save API service


To test this:
- Go to any activity page that we've developed (GSC, Mobile Fuel, Pulp & Paper..)
- Enter data & save
- Check in the DB that at least one record was created in the erc.report_emission_emission_categories table
- Verify using the Emission Summary excel sheet that it is the correct category/categories
